src/snaskii.c

```diff
@@ -100,6 +102,26 @@ void init_game()
 	memset(energy_blocks, INACTIVE_BLOCK, ENERGY_BLOCKS_SIZE * sizeof(coord_t));
 }
 
+
+void tick_step()
+{
+    update_times(&times);
+
+    double sync = (double) MIN_DELAY / get_fps(&times);
+
+    move_snake(&snake, sync);
+}
+
+
+void render(scene_t scene)
+{
+    draw_background((char**) scene);
+    draw_snake(&snake);
+    screen_show();
+    draw_menu(&times);
+}
+
```

Comment:
It would be great to comment what these new functions do.

Reply:
Done!